assets/section-footer.css

```diff
@@ -245,6 +256,13 @@
   color: rgba(var(--color-foreground), 0.75);
 }
 
+@media screen and (max-width: 749px) {
+  localization-form,
+  .localization-form {
+    width: 100%;
+  }
+}
```

Comment:
Wondering if we should keep this or why we need it 🤔 
[Video](https://screenshot.click/22-09-yr607-cnnpt.mp4)